### ISSUE TITLE
Suggestion for improving prize-giving

### ISSUE THREAD

**Issue by PeterJCLaw** [sourcebots/tasks#204]
Not sure how best to submit suggestions, so please do move this elsewhere if there's a better home.

During the prize giving there seemed to be a lot of faff around giving the actual prizes to the participants. Concretely: three volunteers were needed to hand out a number of small prizes to each team and these seemed to get routed via the main speaker.

While it's sort-of nice to have everything handed over by the speaker, it ended up looking rather messy and it might look better if the speaker instead handed over a single item and then the participants simply picked up the other items from a table.

In that regard, it would be good if we could have the prizes in suitable piles on a table on the stage ahead of time. Since the prizes aren't named, this could be arranged well in advance and covered with a cloth or something until needed. These stacks would also be easier for the participants to carry, though if that's a concern we could also explore putt them into goody-bags of some sort.

This would reduce the volunteers needed to a single one (other than the MC) behind the table who pushed forwards the stacks/bags/whatever for the current pile.

I expect this would be simpler for participants and the speaker as well as smoother & faster overall.

**Comment by trickeydan** [comment]
I agree that having a table on the stage would be better, and I don't think it's necessary for any of the prizes to be routed via the guest speaker.
